test/tests/pbs_Rrecord_resources_used.py

```diff
@@ -0,0 +1,616 @@
+        m = self.server.accounting_match(msg='.*Resource_List.*', id=jid1, regexp=True)
+        self.assertNotEqual(m, None)
+        m = self.server.accounting_match(msg='.*R;' + jid1 + '.*resources_used.*', id=jid1, regexp=True)
+        self.assertIsNone(m)
+        m = self.server.accounting_match(msg='.*Resource_List.*', id=jid2, regexp=True)
+        self.assertNotEqual(m,None)
+        m = self.server.accounting_match(msg='.*R;' + jid2 + '.*resources_used.*', id=jid2, regexp=True)
+        self.assertIsNone(m)
+        m = self.server.accounting_match(msg='.*Resource_List.*', id=jid3s1, regexp=True)
+        self.assertNotEqual(m, None)
+
+        m = self.server.accounting_match(msg='.*R;' + re.escape(jid3s1) + '.*resources_used.*',
+        id=jid3s1, regexp=True)
+        self.assertIsNone(m)
+        #sleeping for 30 seconds
+        time.sleep(30)
+        # Rerun the job
+        print " qrerun job first time\n"
+
+        jids =  self.server.select(extend='T')
+        try:
+            self.server.rerunjob(jids)
+        except PbsRerunError:
+            pass
+
+        """
+        Verify that the accounting logs have R logs with last known resource usage Resource_used and
+        run_count should be 2 for J1, No R logs for J2.
+        """
+        self.server.expect(JOB, {ATTR_state: 'R', 'run_count': 2}, jid1, offset=5)
+        self.server.expect(JOB, {ATTR_state: 'R', 'run_count': 1}, jid2)
+        self.server.expect(JOB, {ATTR_state: 'R', 'run_count': 1}, jid3s1)
+
+        m = self.server.accounting_match(msg='.*R;' + jid1 + '.*Exit_status=-11.*.*resources_used.*.*run_count=1.*', id=jid1, regexp=True)
+        self.assertNotEqual(m, None)
+        m = self.server.accounting_match(msg='.*R;' + jid2 + '.*resources_used.*', id=jid2, regexp=True)
+        self.assertIsNone(m)
+        m = self.server.accounting_match(msg='.*R;' + re.escape(jid3s1) + '.*Exit_status=-11.*.*resources_used.*.*run_count=1.*', id=jid3s1, regexp=True)
+        self.assertNotEqual(m, None)
+
+        time.sleep(20)
+        # Rerun the job again
+        print "qrerun job second time\n"
+
+        jids =  self.server.select(extend='T')
+
+        try:
+            self.server.rerunjob(jids)
+
+        except PbsRerunError:
+            pass
+        """
+        Verify that the accounting logs should R logs with last known resource usage Resource_used and
+        run_count should be 3 for J1 , No R logs in accounting for J2
+        """
+        self.server.expect(JOB, {ATTR_state: 'R', 'run_count': 3}, jid1, offset=5)
+        self.server.expect(JOB, {ATTR_state: 'R', 'run_count': 1}, jid2)
+        self.server.expect(JOB, {ATTR_state: 'R', 'run_count': 1}, jid3s1)
+        m = self.server.accounting_match(msg='.*R;' + jid1 + '.*Exit_status=-11.*.*resources_used.*.*run_count=2.*', id=jid1, regexp=True)
+        self.assertNotEqual(m, None)
+        m = self.server.accounting_match(msg='.*R;' + jid2 + '.*resources_used.*', id=jid2, regexp=True)
```

Comment:
Will it be a problem that the accounting log records are accumulating throughout the test?  If you expect multiple R records, how do you know which one you are matching?